tests/handlers/test_password_providers.py

```diff
@@ -106,7 +111,41 @@ def test_password_only_auth_provider_login(self):
         mock_password_provider.reset_mock()
```

Comment:
This line isn't necessary if we're resetting in `setUp`.

Reply:
indeed. removed.